src/lib/characterPreview/summary/DpsScoreGradeRuler.tsx

```diff
@@ -15,16 +17,22 @@ const liftedGrades: Record<string, boolean> = {
   'F+': true,
 }
 
+const reversedLanguages: Languages[] = []
```

Comment:
Probably doesnt matter in the long run for how many languages we have, but if you're doing lookups use a Record<Language, boolean>